public/chat-groups-section.html

```diff
@@ -0,0 +1,56 @@
+<div class="groupsSection">
+    <div class="groupSearch">
+        <h3>Search for groups</h3>
+        <form id="group-search-form" class="groupSearchForm">
+            <input id="group-search-form-input" placeholder="Search for..." class="groupSearchFormInput">
+            <input id="group-search-form-button" type="submit" value="Search" class="button groupSearchFormButton">
+        </form>
+        <div class="groupSearchResults">
+            <ul>
+                <li class="groupSearchItem">
+                    Sample search result <a href="">(View)</a>
+                </li>
+            </ul>
+        </div>
+    </div>
+    <div class="verticalDivider"></div>
+    <div class="groupDetails">
+        <div class="groupInfo">
+            <h1>Group name</h1>
+            <div class="groupDescription">
+                A description of the group being viewed appears here, which may feature some of the group owner's favourite
+                latin texts, such as:
+                Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex
+                ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+                fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt
+                mollit anim id est laborum.
+            </div>
+        </div>
+        <div class="groupOptions">
+            <!-- These items only visible if user is not a member of the group -->
```

Comment:
This is basically just an outline without logic at the moment; I wasn't sure whether to push this, but I figured adding the controller logic and then linking it to the document would probably be a very large pull, even without the service logic to make it complete. With that in mind I figured it'd be a better idea to split it into several parts, the first one being this. If you think it'd be a better idea to include some basic functionality in this branch then I can push some